### DISCUSSION TITLE
How to set environment variables such as "OMP_NUM_THREADS","TF_INTRA_OP_PARALLELISM_THREADS"

### DISCUSSION THREAD

**Discussion by h840473807** [deepmodeling/deepmd-kit#2078] [Q&A] [answered]
Dear all:
  When using dp train, there are sevel warnings that I have not set the environment variables OMP_NUM_THREADS, TF_INTRA_OP_PARALLELISM_THREADS,  TF_INTER_OP_PARALLELISM_THREADS, KMP_BLOCKTIME and KMP_AFFINITY
How can I set these environment variables?  Do these variables affect computation speed? I perform a dp train task with a GPU bound with 6 CPUs.
  Thank you.

**Comment by njzjz** [comment, marked as answer]
You can refer Intel documentation https://www.intel.com/content/www/us/en/developer/articles/technical/maximize-tensorflow-performance-on-cpu-considerations-and-recommendations-for-inference.html
